Keas.Mvc/ClientApp/components/Spaces/SpacesContainer.tsx

```diff
@@ -36,7 +36,7 @@ export default class SpacesContainer extends React.Component<{}, IState> {
     }
 
     public async componentDidMount() {
-        const spaces = await this.context.fetch(`/api/${this.context.team.slug}/spaces/list?orgId=ADNO`);
+        const spaces = await this.context.fetch(`/api/${this.context.team.slug}/spaces/list?teamSlug=${this.context.team.slug}`);
```

Comment:
@jcubbage the slug is already in the URL why would you pass in twice or do I not want to know?